pkg/ride/constructors.gen.go

```diff
@@ -1405,19 +1462,11 @@ func invokeScriptTransactionV4Constructor(_ environment, args_ ...rideType) (rid
 		return nil, errors.Errorf("invokeScriptTransactionV4Constructor: unexpected type '%s' for args", args_[3].instanceOf())
 	}
 	// checks for list elements
-	for _, elem := range args {
-		switch t := elem.(type) {
-		case rideList:
-			for _, elem := range t {
-				switch t := elem.(type) {
-				case rideBoolean, rideByteVector, rideInt, rideString:
-				default:
-					return nil, errors.Errorf("invokeScriptTransactionV4Constructor: unexpected type '%s' in args list", t.instanceOf())
-				}
-			}
-		case rideBoolean, rideByteVector, rideInt, rideString:
+	for _, el := range args {
+		switch el.(type) {
+		case rideBoolean, rideByteVector, rideInt, rideList, rideString:
 		default:
-			return nil, errors.Errorf("invokeScriptTransactionV4Constructor: unexpected type '%s' in args list", t.instanceOf())
+			return nil, errors.Errorf("invokeScriptTransactionV4Constructor: unexpected type '%s' in args list", el.instanceOf())
 		}
```

Comment:
Why `rideList` case checks has been removed? Looks weird.

Reply:
Don't know, it's generated :) 
I'll check why it was here, may be generation broken somehow.